docker-compose.prod.yml

```
@@ -1,10 +1,10 @@
 services:
   app:
-#        build:
-#          context: .
-#          dockerfile: docker/dockerfile/Dockerfile
-#          target: prod
-    image: portabase/portabase:1.7.1
+#    build:
```

YAMLlint (1.38.0)

[warning] comment not indented like content

(comments-indentation)